radis/lbl/factory.py

```diff
@@ -1716,7 +1719,7 @@ def _is_at_equilibrium():
                 raise NotImplementedError("broadening_method not implemented")
         elif optimization is None:
             if broadening_method == "voigt":
-                estimated_time = 6.6487e-08 * n_lines * truncation / wstep * factor
+                estimated_time = 6.6487e-08 * n_lines * truncation / wstep
```

Comment:
is "factor" from L1677 still used anywhere else?

Reply:
No. It is being used by optimization =`simple` and `min-RMS`. For non-eq cases factor = 2 so we can simply refactor it as :
```
if not  _is_at_equilibrium:
   factor = 2
```
and remove rest of the code